### DISCUSSION TITLE
При локальном подключении карты, падает стилизация страницы в момент загрузки

### DISCUSSION THREAD

**Discussion by Denisoov** [yandex-maps-unofficial/vue-yandex-maps#277] [Q&A (2.x)]
Существует страница которая рендерится в ssr.  Есть несколько объявленных компонентов, один из которой является карта, подключение карты находится в самом же дочернем компоненте с такой конструкцией в `setup` 

```
import { createYmapsOptions, VueYandexMaps } from 'vue-yandex-maps';
createYmapsOptions({ apikey });
```

Как можно исправить подобное поведение? Пробовал вынести компонент карты в <client-only> ничего не помогло

**Comment by daniluk4000** [comment]
Создал https://github.com/yandex-maps-unofficial/vue-yandex-maps/issues/278

Такое подключение, как у вас (без плагина) не подразумевалось

**Comment by daniluk4000** [comment]
Обновление выпущено

Я вообще проверил, у меня CSS даже без подключения как плагин прекрасно работает. Не знаю, что у вас такое, но теперь возможность ручного импорта есть

**Reply by Denisoov** [reply]
@daniluk4000 а проверяли совместно с ssr страницей? у меня такая проблема заключалась в директиве v-if. Если на дочерний компонент ( внутри которого компоненты модуля ) навесить директиву `v-if`, то загрузка компонента будто блокирует подгрузку css файлов страницы ( точно не выяснил ). 
С `v-show` работает хорошо. 

И опять же, у меня подключение ( без плагина ) в компоненте `setup`  взято с доки. Но есть опять же есть проблема, в том, что невозможно реактивно отследить `loadError`, только если его обернуть в `watch`

**Reply by daniluk4000** [reply]
Да, проверял, но без v-if. Импортните CSS карты глобально в любом компоненте, раз уж на то пошло, он же крохотный. 

И вы всегда можете изменить способ подключения на initializeOn never и использовать функцию initYmaps.

У меня вообще CSS импортируется при любом импорте из vue-yandex-maps, даже если я не  подключил ни одного компонента. И это так задумано

**Reply by Denisoov** [reply]
@daniluk4000 purgecss у меня видимо блокирует вызов этого файла. Хорошо спасибо
